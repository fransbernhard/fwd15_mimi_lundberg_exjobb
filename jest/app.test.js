import React, { Component } from 'react';
import renderer from 'react-test-renderer';
import App from '../src/app';

jest.mock("react-dom", () => ({ render: () => jest.fn() }));

describe('components', () => {
  describe('<App/>', () => {
    it('renders correctly', () => {
      const tree = renderer
        .create(
          <App />
        )
        .toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
