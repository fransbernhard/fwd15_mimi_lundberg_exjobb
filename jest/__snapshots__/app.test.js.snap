// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components <App/> renders correctly 1`] = `
<div>
  <div
    className="loaderSmall"
  >
    <img
      className="loadingImg"
      src="tits.gif"
      width="400"
    />
  </div>
  <div>
    <div
      className="background-img"
    />
    <header>
      <nav>
        <ul>
          <li>
            <a
              href="#top"
              onClick={[Function]}
            >
              Home
            </a>
          </li>
          <li>
            <a
              href="#about"
              onClick={[Function]}
            >
              About
            </a>
          </li>
          <li>
            <a
              href="/archive"
              onClick={[Function]}
              style={Object {}}
            >
              Archive
            </a>
          </li>
          <li
            className="contactMobile"
          >
            <a
              href="mailto:magdamargaretha@gmail.com?Subject=magdenmagden"
            >
              Contact
            </a>
          </li>
          <li
            className="contactWeb"
          >
            <a
              href="#contact"
              onClick={[Function]}
            >
              Contact
            </a>
          </li>
        </ul>
      </nav>
    </header>
    <div
      className="container"
      id="top"
    >
      <div
        className="hero"
      >
        <div
          className="news"
        >
          <h1>
            V E R N I S A G E
          </h1>
          <p>
            Vernisage @ Medborgarplatsen hos supercoola 
            <a
              href="http://layered.se/"
              target="_blank"
            >
              Layered Official
            </a>
             
            <br />
            Kom drick klappa djuren, säg något kul
          </p>
        </div>
      </div>
      <div
        className="about"
        id="about"
      >
        <div
          className="wrapper-about"
        >
          <div
            className="wrapper-a"
          >
            <img
              className="mySlides about-img"
              src="magden.jpg"
            />
            <p>
              Magda har massa csn lån och nästan en kandidat  
              <br />
              i företagsekonomi. För daglig uppdatering --&gt;
              <br />
               
              <a
                href="https://www.instagram.com/magdenmagden/?hl=sv"
                target="_blank"
              >
                @magdenmagden
              </a>
            </p>
            <br />
          </div>
          <div
            className="wrapper-cv"
          >
            <h5>
              2017
            </h5>
            <p>
              Vernisage @ Layered Official | Stockholm 
              <br />
              Försäljning genom Young Art | Stockholm 
              <br />
              Utställning Nividas/Adornment | Stockholm
            </p>
            <br />
            <h5>
              2016
            </h5>
            <p>
              Vernissage DÅLIG SIKT | Stockholm 
              <br />
              Utställning | L´homme rouge, Stockholm
              <br />
              Artwork "Dying for you" | Otto Knows
            </p>
            <br />
            <h5>
              2015
            </h5>
            <p>
              Vernissage STRÖSSEL | Stockholm
              <br />
              Utställning | Kaffebaren, Malmö
              <br />
              Utställning | Scandic hotel, Malmö
              <br />
              Artwork "Next to me" | Otto Knows
            </p>
            <br />
            <h5>
              2014
            </h5>
            <p>
              Vernissage BOKSLUT | Stockholm
              <br />
              Utställning | Brillo, Stockholm
              <br />
              Utställning | Mood gallerian, Stockholm
              <br />
              Utställning | Höganäs konsthall, Höganäs
            </p>
            <br />
            <h5>
              2013
            </h5>
            <p>
              Vernissage TORSK | Stockholm
              <br />
              Utställning | East, Stockholm
            </p>
            <br />
            <p>
              2011-2012 Lunds Konst och Designskola | Lund
              <br />
              2012-2013 Beckmans kvällsskola | Stockholm
            </p>
          </div>
        </div>
      </div>
      <div
        className="contact"
        id="contact"
      >
        <div
          className="filter"
        >
          <form
            className="form"
            id="formContact"
            onSubmit={[Function]}
          >
            <input
              id="formEmail"
              name="formEmail"
              onChange={[Function]}
              placeholder="email"
              required={true}
              type="email"
              value=""
            />
            <textarea
              cols="40"
              id="formMsg"
              name="formMsg"
              onChange={[Function]}
              placeholder="meddelande"
              required={true}
              rows="8"
              value=""
            />
            <input
              className="btn--cta"
              id="btn-submit"
              type="submit"
              value="Skicka"
            />
          </form>
        </div>
      </div>
      <footer>
        <a
          href="mailto:magdamargaretha@gmail.com"
        >
          magdamargaretha@gmail.com, 
        </a>
        <a
          href="tel:+46735139012"
        >
          +46735139012
        </a>
        <p>
          Kristianstadsgatan 7, 334 56 Malmö
        </p>
      </footer>
    </div>
  </div>
</div>
`;
